bin/create-image.sh

```diff
@@ -4,21 +4,9 @@ set -x
 . ./bin/common.sh
 . ./bin/cloud-common.sh
 
-get_image_name(){
-    DIST=$1
-    echo "$DIST-daily-amd64"
-}
-
-setup_jenkins_home(){
-    local IP=$1
-    local JENKINS_HOME=$2
-    local OPENSTACK_CREDENTIALS_DIR=$3
-    execute_remote_command "$IP" "sudo rm -rf $JENKINS_HOME && \
-sudo mkdir -p $JENKINS_HOME && \
-sudo chmod a+rwx $JENKINS_HOME && \
-sudo chown ubuntu:ubuntu $JENKINS_HOME && \
-mkdir -p $OPENSTACK_CREDENTIALS_DIR"
-}
+PRIVATE_KEY_PATH=${1:-~/.canonistack/${OS_USERNAME}_${OS_REGION_NAME}.key}
+KEYPAIR_NAME=${OS_USERNAME}_${OS_REGION_NAME}
+IMAGE_NAME="uci/cloudimg/$DIST-amd64.img"
```

Comment:
In order to reduce the amount of shell script needed to determine the name of the image to be used (the daily ones carry a timestamp) I was looking for images with fixed names and found this ones, they are (or were) used by the Ubuntu CI Engine team for running tests before "landing new code in the distro" (from the project page). See [1] and [2] 

[1] http://bazaar.launchpad.net/~canonical-ci-engineering/uci-engine/trunk/view/head:/britney_proxy/README.images#L38
[2] http://bazaar.launchpad.net/~canonical-ci-engineering/uci-engine/trunk/view/head:/ci-utils/ci_utils/image_store/__init__.py#L95